src/labels/CustomLabels.labels

```diff
@@ -12691,4 +12691,124 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
         <shortDescription>stgValidationHHAccountHHRules</shortDescription>
         <value>The Account Model is set to &apos;Household Account&apos;, which requires Household Rules to be set to &apos;No Contacts&apos;.   When using Household Accounts, there is no need to have an additional Household Object.</value>
     </labels>
+     <labels>
+        <fullName>gsResourcesTitle</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>The page title in the Getting Started Resources section</shortDescription>
+        <value>Resources</value>
+    </labels>
+    <labels>
+        <fullName>gsResourcesItemNonprofitCloudLabel</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource NonprofitCloud label in Getting Started Resources section</shortDescription>
```

Comment:
Need a space between Nonprofit and Cloud. Also curious why the descriptions all start with Resource? I don't think you need it since you say that the labels are in the Getting Started Resources section.